classes/Announcements.php

```diff
@@ -49,7 +49,13 @@ class Announcements {
 	 * @return void
 	 */
 	public function __construct() {
-		$this->page_title = __( 'Announcements', 'tutor' );
+		add_action(
```

Comment:
Instead of 6 line of code, we can write it in on-liner
```php
add_action( 'init', fn() => $this->page_title = __( 'Announcements', 'tutor' ) );
```

Reply:
We can not do this as per this warning: **Assignments must be the first block of code on a line**